yarn.lock

```diff
@@ -18639,13 +18639,6 @@ p-map@^3.0.0:
   dependencies:
     aggregate-error "^3.0.0"
 
-p-map@^4.0.0:
-  version "4.0.0"
-  resolved "https://registry.yarnpkg.com/p-map/-/p-map-4.0.0.tgz#bb2f95a5eda2ec168ec9274e06a747c3e2904d2b"
-  integrity sha512-/bjOqmgETBYB5BoEeGVea8dmvHb2m9GLy1E9W43yeyfP6QQCZGFNa+XRceJEuDB6zqr+gKpIAmlLebMpykw/MQ==
-  dependencies:
-    aggregate-error "^3.0.0"
-
```

Comment:
curious what happened here

Reply:
Not sure what happened here or why it was included

Reply:
When I ran `yarn` locally this actually got pulled back in. You might want to check on this. FWIW I am using Node 14 and yarn 1.21.1

Reply:
Same. @shawyu-okta mind running `yarn why`? I had it a minute ago but had to switch back to my branch. `p-map` was a dependency of a few libs we use.

Reply:
Yep, i have the same issue, but I have just been ignoring the yarn.lock file with my commits 😅

Reply:
It looks like it's coming from testcafe: 
```
$ yarn why p-map
yarn why v1.21.1
[1/4] 🤔  Why do we have the module "p-map"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "p-map@4.0.0"
info Has been hoisted to "p-map"
info Reasons this module exists
   - "workspace-aggregator-33de0f1f-f67a-4907-a1f7-4fa47e340881" depends on it
   - Hoisted from "_project_#cacache#p-map"
   - Hoisted from "_project_#v3#lint-staged#listr2#p-map"
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "40KB"
info Disk size with transitive dependencies: "80KB"
info Number of shared dependencies: 3
=> Found "testcafe#p-map@1.2.0"
info Reasons this module exists
   - "_project_#testcafe#del" depends on it
   - Hoisted from "_project_#testcafe#del#p-map"
info Disk size without dependencies: "16KB"
info Disk size with unique dependencies: "16KB"
info Disk size with transitive dependencies: "16KB"
info Number of shared dependencies: 0
=> Found "testcafe-browser-tools#p-map@3.0.0"
info Reasons this module exists
   - "_project_#testcafe#testcafe-browser-tools#del" depends on it
   - Hoisted from "_project_#testcafe#testcafe-browser-tools#del#p-map"
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "40KB"
info Disk size with transitive dependencies: "80KB"
info Number of shared dependencies: 3
=> Found "v3#p-map@4.0.0"
info Reasons this module exists
   - "_project_#v3#preact-cli#compression-webpack-plugin#cacache" depends on it
   - Hoisted from "_project_#v3#preact-cli#compression-webpack-plugin#cacache#p-map"
   - in the nohoist list ["/_project_/v3/ajv-errors","/_project_/v3/@types/ajv-errors","/_project_/v3/preact-cli","/_project_/v3/preact-cli/**","/_project_/v3/preact-cli-plugin-env-vars","/_project_/v3/preact-cli-plugin-env-vars/**","/_project_/v3/preact-render-to-string","/_project_/v3/preact-render-to-string/**","/_project_/v3/preact-svg-loader","/_project_/v3/preact-svg-loader/**","/_project_/**/@wdio/**","/_project_/**/@types/ajv-errors","/_project_/**/@types/ajv-errors/**"]
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "40KB"
info Disk size with transitive dependencies: "80KB"
info Number of shared dependencies: 3
✨  Done in 1.56s.
```

Reply:
https://github.com/okta/okta-signin-widget/pull/2617/commits/ed0b36b98b93de74cfaf418e835872476111dd3a
it comes from this commit, not sure why it's in this PR ...

Reply:
We can investigate later. Can we update the lock file for now?

Reply:
lock file has been updated